readme-template.md

```diff
@@ -1,106 +1,93 @@
+* [Deploying](#deploying)
+* [Contributing Rules](#contributing-rules)
+* [Common Questions](#common-questions)
 
-<a name="documentation"></a>
 ## Further Documentation
 
+<!-- Examples of some documentation to link from the readme -->
 * **[System Roadmap]()**
 * **[Regression Testing Checklist]()**
 * **[Change Log]()**
 
 Include any other relevant documentation in this section.
 
-<a name="overview"></a>
 ## System Overview
 
 * More in-depth overview of the system, what this is trying to accomplish and how it achieves its goal.
 * Keep it concise, write this section like the first paragraph of a blog post, what is the point of this repo?
+* Perhaps label some of the core tech used in the repo and link to the external websites/documentation
 
-<a name="installation"></a>
 ## Installation
 
-* Recordit of installation setup for new users
 * Step-by-step guide on installation
 
-<a name="running"></a>
 ## Running Locally
 
-* Recordit of setting up the app for running locally
 * Step-by-step guide of setting up locally
 
-<a name="testing"></a>
-## Testing
-
-* What CI do the test run on?
-* What testing framework is used?
-
-<a name="cheat-sheet"></a>
 ## Script Cheat Sheet
 
 * List scripts used in the repo and a description of what they do
 * Example: `npm run build` := Generates `dist` folder with changes made in `lib`
 
-<a name="testing"></a>
 ## Testing
 
-<a name="automated"></a>
+* What CI do the test run on?
+* What testing framework is used?
+* How are the tests structured in this repo? Is there a strict writing style to follow?
+
 ### Automated Testing
 
-* Step-by-step to run unit/selenium tests
+* Step-by-step to run all automated testing.
+* What are some of the commands I need to know when running test in my local environment?
+* Any tips on running singular tests or groups of tests at once in this repo?
 
-<a name="manual"></a>
 ### Manual Testing
 
 * Link to external regression checklist document
+* If anything particular is setup for this repo, describe how you would manually test a feature branch before going to production
 
-<a name="built-with"></a>
 ## Built With
 
 * What frameworks are used? Links to framework API docs.
 * Essential node modules used, links to documentation/repo.
 
-<a name="deploy"></a>
-## Deploying
-
-* How do deploys work for new pull requests?
-* What is the process from a feature branch to production?
-
-<!-- Optional on README but should be linked externally if not -->
-<a name="linting"></a>
 ## Linting
 
 * How to I lint specific file types?
 * Do these run via git hooks
 * Link to linting rules
 
-<a name="rules"></a>
+## Deploying
+
+* How do deploys work for new pull requests?]
+* Steps on how to get a feature branch merged and deployed into production environment
+
 ## Contributing Rules
 
 * Link to HX-culture documentation on contribution
 * Any specific rules for this repo?
 
-<a name="faq"></a>
 ## Common Questions
 
 * Frequently asked questions that will help engineers
```

Comment:
Possibly good to add in "speak to the relevant guild/pod for any other questions" like we have on the Render and Tripapp ReadME